backend/src/services/summary/index.ts

```diff
@@ -10,12 +11,30 @@ import { getCountsCached, getUsageStatsCached } from "./usageStats"
 export const getSummaryData = async (
   ctx: MetloContext,
 ): Promise<SummaryResponse> => {
+  const startTopEndpoints = performance.now()
```

Comment:
can you put these inside, this is the cached stuff

Reply:
Yes, can do.